tf2onnx/onnx_opset/nn.py

```diff
@@ -538,8 +540,9 @@ def version_7(cls, ctx, node, **kwargs):
         new_line = g.make_node(op_type="Concat", inputs=[const_zero_bool.output[0], "line"],
                                attr={"axis": counter_axis},
                                dtypes=[onnx_pb.TensorProto.BOOL])
-        slice_node = g.make_node(op_type="Slice", inputs=[new_line.output[0]],
-                                 attr={"axes": [counter_axis], "starts": [0], "ends": [-1]})
+        attr = {"axes": [counter_axis], "starts": [0], "ends": [-1]}
+        inputs_map = {"data": new_line.output[0], **attr}
+        slice_node = g.make_slice_node(inputs_map)
```

Comment:
graph class may not be a good place for make_xxx. we may add more make_xxx in the future.

i am considering to introduce a builder class.
similar to https://github.com/nbcsm/tensorflow-onnx/blob/builder/tf2onnx/graph.py#L894

builder.make_xxx is mid-level api, and wrap one or more graph.make_node inside.

```
builder = Builder(g)
builder.set_scope("my_scope") # optional
builder.make_slice(name="my_slice", {data=xx, axes=xx})
```